compute/admin_guide/compliance/detect_secrets.adoc

```diff
@@ -55,10 +56,87 @@ image::detect_secrets_762507.png[width=800]
 Set the action to *ALERT* instead of *BLOCK*, then go to *Monitor > Compliance* after running the image.
 Click on the image under *Images* tab.
 
-image::detect_secrets_762508.png[width=800]
+==== Compliance Check ID 597
 
+This check detects sensitive information provided in environment variables of container.
 
-[.section]
-=== Compliance check ID 597
+[#agentless-secret-detection]
+=== Agentless Secret Detection
 
-This check detects sensitive information provided in environment variables of container.
+Agentless scanning detects sensitive information inside your hosts and container images.
```

Comment:
Agentless scanning detects sensitive information inside files in your hosts and container images filesystem.